testsuite/tests/ppx-contexts/Makefile

```diff
@@ -0,0 +1,41 @@
+#**************************************************************************
+#*                                                                        *
+#*                                OCaml                                   *
+#*                                                                        *
+#*                              Peter Zotov                               *
```

Comment:
This, and the copyright notice below, do not seem right.

Reply:
Ouch.

Reply:
My understanding is:
- If this Makefile was pilfered from somewhere else, you should preserve the original author attribution, and add your name and copyright if you made non-trivial modifications.
- If this Makefile is your own work, then it should have your name and copyright (and no-one else's to begin with).

In both cases they should have the LGPL paragraph.

I'm suspecting something along these lines didn't happen correctly at a previous time, since this Makefile has an INRIA copyright yet a non-INRIA personal affiliation.

Reply:
There is no copyright header in the testsuite tests themselves.  @mshinwell  Do you know why we keep them in the testsuite Makefile?

Reply:
I do not.

Reply:
@damiendoligez @xavierleroy What is the official recommendation w.r.t. copyright headers in the testsuite?  Currently they are kept in Makefiles but not in tests themselves.  Should we get rid of them in Makefiles as well (or at least not require them for new Makefiles)?

Reply:
No copyright headers in the test suite except for test infrastructure scripts and Makefiles. In general, the test directories themselves should not have any headers, even in Makefiles. I just haven't gotten around to remove them yet (and I won't need to thanks to ocamltest).

Reply:
Ok, so @camlspotter : can you just remove the header?